servermanagement/startSignal.go

```diff
@@ -15,13 +15,24 @@ import (
 func StartSignal(config types.Configuration, RaftServers map[string]types.RaftServer) error {
 	wg := &sync.WaitGroup{}
 	wg.Add(len(config.Servers))
-	designationMap := make(map[string]string)
+	designationMap := make(map[string]types.State)
+	state := types.State{
```

Comment:
`types.State` will likely have many more functions related to. Let's create a new file for it and write a function `BuildInitialState()`